### ISSUE TITLE
Renumbering Clusters after Sorting

### ISSUE THREAD

**Issue by seacritasianman** [klusta-team/klustaviewa#66]
Hello,

I was wondering if there's a way to renumber all the clusters after sorting, either manually, or automatically (starting with 0 being the noise clusters to N-1 clusters). Thanks

**Comment by rossant** [comment]
It's not possible at the moment, unfortunately.

**Comment by seacritasianman** [comment]
Do you know if there's any easy way to find the cluster ID of the noise and MUA clusters after using Klustaviewa, esp via MATLAB? Thanks.

**Comment by rossant** [comment]
Yes, open the kwik file in MATLAB and check out the path `/channel_groups/0/clusters/main/47/` (for cluster 47 of shank 0), the attribute name is `cluster_group`, the value is 0 or 1 for noise or MUA. See https://www.mathworks.com/help/matlab/ref/h5readatt.html?s_tid=gn_loc_drop

**Comment by seacritasianman** [comment]
Perfect. Thank you for your help.
